class01/homework/ParkYoungMoon/02_camera.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import cv2
+# Read from the first camera device
+cap = cv2.VideoCapture(0)
+
+topLeft = (50, 50)
+bottomRight = (300, 300)
+center = (150, 150)
+def on_mouse(event, x, y, flags, param):
+    if event == cv2.EVENT_LBUTTONDOWN:
+        cv2.circle(frame,(x,y),25,(255,0,0),-1)
+        cv2.imshow("Camera",frame)
+# 성공적으로 video device 가 열렸으면 while 문 반복
+while(cap.isOpened()):
+    # 한 프레임을 읽어옴
+    ret, frame = cap.read()
+    if ret is False:
+        print("Can't receive frame (stream end?). Exiting ...")
+        break
+    
+    
+    # Line
+    cv2.line(frame, topLeft, bottomRight, (0, 255, 0), 5)
+
+    # Rectangle
+    cv2.rectangle(frame, 
+        [pt+30 for pt in topLeft], [pt-30 for pt in bottomRight], (0, 0, 255), 5) 
+
+    # Text 
+    font = cv2.FONT_HERSHEY_SIMPLEX
+    cv2.putText(frame, 'ddd', [pt+80 for pt in topLeft], font, 2, (0, 255, 255), 10)
+    font=cv2.FONT_ITALIC
+    cv2.putText(frame, 'PYM', center, font, 5, (255,0,200), 10)
+
+
+    # Display
+    cv2.imshow("Camera",frame)
+    
+    cv2.setMouseCallback('Camera',on_mouse,frame)
+    # 1 ms 동안 대기하며 키 입력을 받고 'q' 입력 시 종료
+    key = cv2.waitKey(1)
+    if key & 0xFF == ord('q'):
+        break
+

```

Comment:
To add a circle and to modify a circle location when mouse click, refer to the below functions please.
- cv2.circle()
- cv2.setMouseCallback()